### ISSUE TITLE
Update jimtcl

### ISSUE THREAD

**Issue by EvanTheB** [raspberrypi/openocd#26]
Jimtcl has a new home: https://github.com/msteveb/jimtcl

And recently fixed the whitespace compilation issue: https://github.com/msteveb/jimtcl/commit/bcd4434161897da5203cdfaef8b2227e15d82c45

**Comment by lurch** [comment]
Yes, the whitespace issue got fixed thanks to me reporting it :smiley: 

I've been planning to bump the version of `jimtcl` used in this repo, but before I can do that I need to check that openocd, VSCode and picoprobe all still work properly, and unfortunately I've not had time to do that yet... :slightly_frowning_face:

**Comment by lurch** [comment]
(@aallan please feel free to assign this issue to me)

**Comment by lurch** [comment]
@EvanTheB I assume that this issue is now redundant? (gosh, 3 years old!)
